Casks/c/codebolt.rb

```diff
@@ -0,0 +1,32 @@
+cask "codebolt" do
+  arch arm: "arm64", intel: "x64"
+
+  version "1.1.17"
+  sha256 arm:   "216c4fe9d254da61613178f1f9fa896d6c68012cb2c10eb79efe91fe1b43158c",
+         intel: "6631b8187bc4aaa2198eb5af2d8251dae45ac79084cae68a41e96007f3422159"
+
+  url "https://releasebuilds.codebolt.ai/CodeBolt-#{version}#{"-#{arch}" if arch == "arm64"}.dmg"
+  name "CodeBolt"
+  desc "AI Powered Code Editor"
+  homepage "https://codebolt.ai/"
+
+  livecheck do
+    url "https://github.com/codeboltai/codebolt/releases/latest"
+    strategy :github_latest
```

Comment:
```suggestion
    url "https://github.com/codeboltai/codebolt/"
```

The `git` strategy is preferable here unless there's a reason `github_latest` is required. If using `git` we don't need to provide it, as it will be automatically detected by the url.